css/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: poppins;
+  src: url("https://fonts.googleapis.com/css?family=Poppins");
+}
+
+.header {
+  width: 375px;
+  height: 90px;
+  color: #6070ff;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+h2 {
+  font-size: 25px;
+}
+
+.material-icons {
+  margin-top: 25px;
+  font-size: 35px;
+}
+
+.container {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-image: url(header-shapes\ mobile@2x.png);
+  background-repeat: no-repeat;
+  border-radius: 0 0 10px 0;
+}
+
+h1 {
+  font-size: 35px;
+  padding-top: 80px;
+  color: #172b4d;
+}
+
+p {
+  font-size: 16.9px;
+  color: #172b4d;
+}
```

Comment:
- [ ] Please make sure to use the recommended styling here as well. Therefore, please make sure to apply the correct `font-size` and `font-weight`. Also, please make sure to use `#344563` instead of `#172b4d;` for `p`
- [ ] Also, please makes sure to reduce the space between `h1` and `p` according to the given design guidelines

Your design | Given design 
---| ---|
![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/a4aaf5d3-81b7-42d9-8886-c99cf52f9b3f) | ![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/92fad667-bbfc-4cde-8561-d8b8d5255971)